### PR TITLE
Copy todo list code to new repository for code review

### PR DESCRIPTION
**Pull request by raihan2bd**
# Todo-list

- In this PR I copy my to-do list app code and make a new branch.
- Using this project anyone can add, edit, delete, and clear all to-do lists.
- Also user's to-do list will be saved on browser localStorage
- Last but not least I fix the linter error.

# Demo of my project
 demo

### CODE REVIEW

File: src/modules/domSelector.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const formInput = document.getElementById('description');
+export const showMsg = document.querySelector('.show-msg');
+export const btnSubmit = document.querySelector('.btn-submit');
+export const todoContainer = document.querySelector('.todo-items-gropu');
```
Comment (Lookingfor00) [inline src/modules/domSelector.js:4]: Great job creating an external file for your dom Selector! I learned something new! 👏 🥇

File: src/modules/todos.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+import threeDotIcon from '../assets/three-dot-24.png';
+import deleteIcon from '../assets/trash-24.png';
+
+export default class Todos {
+  constructor() {
+    if (localStorage.getItem('todos')) {
+      this.todos = JSON.parse(localStorage.getItem('todos'));
+    } else {
+      this.todos = [];
+    }
```
Comment (Lookingfor00) [inline src/modules/todos.js:14]: Great job loading data from local storage directly in the constructor file! I learned something new 😄 👏 🎖️

File: src/modules/todos.js [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+        showMsg.classList.remove('form-error');
+        showMsg.innerText = '';
+        e.target.classList.remove('invalid-edit');
+      }
+    });
+
+    // todo Delete dot button
+    const deleteTodo = document.createElement('button');
+    deleteTodo.className = 'btn-delete';
+    deleteTodo.innerHTML = `<img src="${deleteIcon}" alt="X">`;
```
Comment (Lookingfor00) [inline src/modules/todos.js:206]: - [x] I suggest you: 
    - either add an explicit value to the `alt` to make the project more accessible ([more detail here](https://www.w3.org/WAI/test-evaluate/preliminary/#images))
    - or replace `` with external icons using links like [font awesome](https://fontawesome.com/icons), which may help when scaling, and changing icons's color. etc...
 
(Please do the same in other places of the file with `alt`)

**Review by Lookingfor00 — APPROVED** [review]
# Approved 🟢 
Hi @raihan2bd, great job, your project is complete! you can merge it! 👏 
Congratulations!
giphy

There are some optional changes that may improve the project! Please consider checking them

### Highlights
- [x] Customizing the UI 👍 
- [x] Clean and Readable code 👏 
- [x] Great job adding the optional feature 🥇 

### Optional Suggestions
- [x] [Optional] Really great job implementing the optional features, please save new task order after the drag
   There is a little bug when we use the drag to change task order, after reloading the page or adding a task to order change again, it would be great if you can fix that.
**how to reproduce the bug:**
Re-orderer tasks by drag
image
Add a new task and all tasks are re-ordered in the previous order (canceling the drag-drop order effect)
image

- [x] [Optional] Please check the optional changes in **inline comment**



Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Comment by raihan2bd** [comment]
Hi @Trast00, Thanks a lot for your suggestions. I will try to update your optional suggestions near future.

**Pull request merged**